corehq/apps/userreports/views.py

```diff
@@ -583,6 +583,29 @@ def dispatch(self, request, *args, **kwargs):
         raise Http404("Report was not created by the report builder")
 
 
+def to_report_column(column):
+    """
+    column is the JSON that we get when saving or previewing a report. Return a column spec we can use to create a
+    ReportConfiguration.
+    """
+    # Example value of column:
+    #     {
+    #         u'column_id': u'modified_on_6457b79c',
+    #         u'name': u'modified_on'
+    #         u'label': u'modified on',
+    #         u'is_numeric': False,
+    #         u'is_group_by_column': False,
+    #         u'aggregation': None,
+    #     }
+    return {
+        'aggregation': column.get('aggregation') or 'simple',
+        'field': column['column_id'],
+        'display': column['label'],
+        'type': 'field',
+        'format': 'default',
+    }
```

Comment:
I know this PR is WIP but noticed this isn't returning a "column_id" so thought I'd just point that out. Current report builder is also adding a transform that rounds decimals. See [`ColumnOption.to_column_dicts()`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/reports/builder/columns.py#L34).